src/Roslyn.Suppressions/Roslyn.Suppressions.UnitTests/SuppressionExecutionContextTests.cs

```diff
@@ -60,14 +60,14 @@ public void SonarProjectKey_PathIsNotValid_ReturnsNull()
         }
 
         [TestMethod]
-        public void SonarProjectKey_HasMixedPaths_ReturnsKey()
+        public void SonarProjectKey_HasMixedPaths_ReturnsExpectedSolutionName()
         {
             var additionalText1 = new ConcreteAdditionalText(@"C:\project\projectKey1\CSharp\SonarLint.xml");
-            var additionalText2 = new ConcreteAdditionalText(@"C:\project\.sonarlint\projectKey2\CSharp\SonarLint.xml");            
+            var additionalText2 = new ConcreteAdditionalText(@"C:\project\SonarLint for Visual Studio\Bindings\expectedSlnName\CSharp\SonarLint.xml");            
```

Comment:
Since we're using AppData it could be more useful for future readers to see real folder structure in some or at least one of these tests

Reply:
Good point. I'll add a note to fix it during a hardening sprint.